dashboard/src/components/Config/ServiceBrokerList/ServiceBrokerItem.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+
+import { CardFooter } from "../../../components/Card";
+import InfoCard from "../../../components/InfoCard";
+import azureIcon from "../../../icons/azure.png";
+import gcpIcon from "../../../icons/gcp.png";
+import placeholder from "../../../placeholder.png";
+import { IServiceBroker } from "../../../shared/ServiceCatalog";
+import SyncButton from "./SyncButton";
+
+interface IServiceBrokerItemProps {
+  broker: IServiceBroker;
+  sync: (broker: IServiceBroker) => Promise<any>;
+}
+
+function getIcon(brokerURL: string) {
+  const parser = document.createElement("a");
+  parser.href = brokerURL;
+  if (parser.hostname.match(/azure/)) {
+    return azureIcon;
+  } else if (parser.hostname.match(/servicebroker\.googleapis/)) {
+    return gcpIcon;
+  }
+  return placeholder;
+}
+
+const ServiceBrokerItem: React.SFC<IServiceBrokerItemProps> = props => {
+  const { broker, sync } = props;
+  return (
+    <InfoCard
+      title={broker.metadata.name}
+      info={`Last updated ${broker.status.lastCatalogRetrievalTime}`}
```

Comment:
It would be great to humanize this in another patch by using moment.js or equivalent.